<?php

namespace ContainerEw6Z4sR;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\ContainerInterface;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getApiErrorControllerreceiveErrorService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.zVKyHiy.App\Controller\ApiErrorController::receiveError()' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.zVKyHiy.App\\Controller\\ApiErrorController::receiveError()'] = (new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService ??= $container->getService(...), [
            'errorTicketRepository' => ['privates', 'App\\Repository\\ErrorTicketRepository', 'getErrorTicketRepositoryService', true],
            'sendMailService' => ['privates', 'App\\Service\\SendMailService', 'getSendMailServiceService', true],
        ], [
            'errorTicketRepository' => 'App\\Repository\\ErrorTicketRepository',
            'sendMailService' => 'App\\Service\\SendMailService',
        ]))->withContext('App\\Controller\\ApiErrorController::receiveError()', $container);
    }
}
